docs/use-cases/image-data.md

```diff
@@ -1,70 +1,128 @@
 ---
-title: "Capture and sync image data"
+title: "How to capture, filter, and sync image data"
 linkTitle: "Capture and sync image data"
 weight: 20
 type: "docs"
+tags: ["data management", "data", "services"]
 images: ["/services/ml/collect.svg"]
-description: "Capture image data from a camera on your machine and sync that data to the cloud."
+description: "Capture images from a camera on your machine and selectively sync images to the cloud with filtering."
 ---
 
-You can use the data management service to capture images from a camera on your machine and sync those images to the cloud.
-Once you have synced your images, you can view them in the Viam app, filter your images using common search criteria, or export them to other machines.
+You can use Viam's built-in data management service to capture images from a camera on your machine and sync the images to the cloud.
 
-For example, you might add the data management service to multiple machines to be able to sync captured images from each of them to the Viam app so that you can search across all images from one interface.
+If you want to capture only certain images, such as those containing a person, you can use a "filtering camera" to selectively capture images based on a computer vision model.
+
+With your images synced to the cloud, you can view images from all your machines in one Viam app interface.
+From there, you can use your image data to do things like [train ML models](/use-cases/deploy-ml/).
+
+<div style="border:1px solid; box-shadow:5px 5px 0 0;padding:1rem; margin:1rem; background-color:#f5f5f5">
+
+**In this page:**
+
+1. [Collect image data and sync it to the cloud](#collect-image-data-and-sync-it-to-the-cloud)
+2. [Use filtering to collect and sync only certain images](#use-filtering-to-collect-and-sync-only-certain-images)
+
+</div>
+
+## Collect image data and sync it to the cloud
 
 {{< table >}}
 {{< tablestep >}}
 {{<imgproc src="/icons/components/camera.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="configure a camera component">}}
-**1. Configure a camera**
+**1. Configure a camera [_(i)_](/components/camera/)**
 
 First, [create a machine](/cloud/machines/#add-a-new-machine) if you haven't yet.
```

Comment:
Can you add the creating a machine as a prerequisite instead please as with the sensor journeys: https://github.com/viamrobotics/docs/pull/2986/files#commit-suggestions.